test/singer_clojure/catalog_test.clj

```diff
@@ -0,0 +1,93 @@
+(ns singer-clojure.catalog-test
+  (:require [clojure.test :refer :all]
+            [singer-clojure.catalog :refer :all]))
+
+(deftest get-selected-streams-test
+  (testing "Test that the get-selected-stream properly returns a sequence
+  of selected streams"
+    (let [catalog {"stream_a" { "stream"       "stream_a"
+                               "metadata"      {"selected" true}}
+                   "stream_b" { "stream"       "stream_b"
+                               "metadata"      {"selected" false}}
+                   "stream_c" { "stream"       "stream_c"
+                               "metadata"      {"selected" true}}
+                   "stream_d" { "stream"       "stream_d"
+                               "metadata"      {"selected" false}}
+                   "stream_e" { "stream"       "stream_e"
+                               "metadata"      {"selected" true}}}]
+      (is (= (get-selected-streams catalog)
+             '("stream_a" "stream_c" "stream_e"))))))
+
+;; `test-serialized-catalog` and `test-deserialized-catalog` are def'd
+;; here since they are also used in the parse tests
+(def serialized-test-catalog {"streams"
```

Comment:
You renamed the method but left the comment referring to the old method name. Maybe just remove the name from the comment?

Reply:
Just went ahead and fixed the names.